packages/devtools_app/lib/src/ui/flutter/service_extension_widgets.dart

```diff
@@ -227,16 +230,50 @@ class _RegisteredServiceExtensionButtonState
   Widget build(BuildContext context) {
     if (_hidden) return const SizedBox();
 
-    return FlatButton(
-      onPressed: _disabled ? null : _click,
-      child: Row(children: [
-        getIconWidget(widget.serviceDescription.icon),
-        if (widget.showTitle)
-          Padding(
-            padding: const EdgeInsets.only(left: 8.0),
-            child: Text(widget.serviceDescription.title),
-          ),
-      ]),
+    return InkWell(
+      onTap: _click,
+      child: Container(
+        constraints: const BoxConstraints(minWidth: 48.0, minHeight: 48.0),
+        alignment: Alignment.center,
+        child: Row(children: [
+          getIconWidget(widget.serviceDescription.icon),
+          if (widget.showTitle)
+            Padding(
+              padding: const EdgeInsets.only(left: 8.0),
+              child: Text(widget.serviceDescription.title),
+            ),
+        ]),
+      ),
+    );
+  }
+}
+
+class HotReloadButton extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return RegisteredServiceExtensionButton(
+      hotReload,
+      () => serviceManager.performHotReload(),
+      (arg) {
```

Comment:
These icons look too colorful. make them black and white as in the devtools dart:html app.
<img width="841" alt="Screen Shot 2019-10-21 at 9 14 33 PM" src="https://user-images.githubusercontent.com/1226812/67258958-0a59f100-f448-11e9-9c65-661d4767e113.png">

Reply:
Done.